src/clarity-angular/datagrid/datagrid-action-overflow.ts

```diff
@@ -61,6 +70,11 @@ export class DatagridActionOverflow {
             //Start checking if current and actionMenuHost are equal. If not traverse to the parentNode and check again.
             while (current) {
                 if (current === actionMenuHost) {
+                    // if this datagrid has checkboxes, uncheck all but this current one
+                    if (this.selection.selectable) {
+                        this.selection.current = [];
+                        this.selection.setSelected(this.item, true);
+                    }
```

Comment:
If you need the model or the row, just inject the `DatagridRow` parent in the constructor, its `item` property is public. No need to ask the user to re-specify it.